### ISSUE TITLE
Yaml extract for routes generate v1 route

### ISSUE THREAD

**Issue by jfdenise** [eclipse/jkube#383]
### Description
Add a yml extract to customize route generation, the generated route is of apiVersion v1 instead of  route.openshift.io/v1
For example, the file route.yml contains:
```
spec:
  tls:
    insecureEdgeTerminationPolicy: Redirect
    termination: edge
  to:
    kind: Service
    name: microprofile-config
```

### Info

* Eclipse JKube version : 1.0.0

* If it's a bug, how to reproduce : add route.yml in src/main/jkube directory. The generated route is of the wrong v1 version.

**Comment by manusa** [comment]
Hi @rohanKanojia  is this issue fixed on #406 ?

**Comment by rohanKanojia** [comment]
Yes, I think #406 should take care of this issue
